sdk/appconfiguration/perf-tests/app-configuration/README.md

```diff
@@ -0,0 +1,9 @@
+### Guide
+
+1. Build the app-config perf tests package `rush build -t perf-app-configuration`.
+2. Copy the `sample.env` file and name it as `.env`.
+3. Create a storage account and populate the `.env` file with `APPCONFIG_CONNECTION_STRING` variable.
+4. Run the tests as follows
+
+   - list settings
```

Comment:
What's special about these settings?

Reply:
list configuration settings? what do you mean?